scripts/mkcloudhost/allocpool

```diff
@@ -28,8 +28,10 @@ foreach my $d (@dirs) {
                 my $signalled=kill(0, $num);
                 diag "PID file had $num";
                 if($signalled) {
-                        diag "someone else owns $a and is alive";
-                        next;
+                        if(`grep mkcloud /proc/$num/cmdline` == 0) {
```

Comment:
I'd prefer allocpool to remain generic and not know about mkcloud. We could achive this using something like
```perl
if(!$ENV{CHECKPROC} || `grep $ENV{CHECKPROC} /proc/$num/cmdline` == 0)
```

Reply:
I agree, that it makes sense to keep allocpool independent - very good point. Let me do some checks with the proposed code.